apps/www/components/Profile/EditProfile/Submit.js

```diff
@@ -27,6 +27,24 @@ const Submit = ({ me, user, t, state, setState, update }) => {
     return null
   }
 
+  // Function to ensure URL has protocol and strip parameters
+  const normalizeUrl = (url) => {
+    // Add protocol if missing
+    const urlWithProtocol = url.indexOf('https://') === -1 && 
+      url.indexOf('http://') === -1
+      ? 'https://' + url
+      : url
+    
+    // Strip parameters
+    try {
+      const urlObj = new URL(urlWithProtocol)
+      return `${urlObj.protocol}//${urlObj.host}${urlObj.pathname}`
+    } catch (e) {
+      // If URL parsing fails, return the URL with protocol
+      return urlWithProtocol
```

Comment:
Are we sure about this? Won't it lead to faulty urls in the DB? How about showing an error message instead?

Reply:
the test if it's a valid url happens in ProfileFieldsUrl.js before the submit. if it's not, the submit function will not even call.